search/query_language_whitebox_test.go

```diff
@@ -270,7 +271,74 @@ func TestParseMap(t *testing.T) {
 		assert.Equal(t, expectedQuery, actualQuery)
 	})
 
+	t.Run(OPTS, func(t *testing.T) {
+		t.Parallel()
+		// given
+		input := fmt.Sprintf(`{"%s": {"parent-exists": true, "tree-view": true}}`, OPTS)
+		// Parsing/Unmarshalling JSON encoding/json
+		fm := map[string]interface{}{}
+		err := json.Unmarshal([]byte(input), &fm)
+		require.Nil(t, err)
```

Comment:
Please use `require.NoError(t, err)` here and below. I have a PR (https://github.com/fabric8-services/fabric8-wit/pull/1813) to fix it everywhere else.